@app/server/src/plugins/SubscriptionsPlugin.ts

```diff
@@ -67,10 +33,25 @@ const currentUserTopicFromContext = async (
  * And see the database trigger function `app_public.tg__graphql_subscription()`.
  */
 const SubscriptionsPlugin = makeExtendSchemaPlugin((build) => {
-  const { pgSql: sql } = build;
+  const currentUserIdSource = build.input.pgSources.find(
+    (s) => s.name === "current_user_id"
+  );
+  if (!currentUserIdSource) {
+    throw new Error("Couldn't find current_user_id source");
+  }
+  const usersSource = build.input.pgSources.find(
+    (s) =>
+      !s.parameters &&
+      s.extensions?.pg?.schemaName === "app_public" &&
+      s.extensions.pg.name === "users"
+  );
```

Comment:
This is an alternative approach if you know the database name rather than the inflected source name. I include it as an alternative to the much simpler equivalent code we saw before:

```ts
const userSource = build.input.pgSources.find((s) => s.name === "users");
```